internal/sci/kind/server.go

```diff
@@ -0,0 +1,110 @@
+package kind
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/hex"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	sci "github.com/substratusai/substratus/internal/sci"
+)
+
+var _ sci.ControllerServer = &Server{}
```

Comment:
what is the reason for doing this? Trying to understand why this is needed

Reply:
Not needed, just creates a compile-time error if the `Server{}` does not satisfy the intended `sci.ControllerServer` interface.